app/config/deletion_counter.yml

```diff
@@ -0,0 +1,8 @@
+parameters:
+  deletion_counter:
+    enwiki:
```

Comment:
I think this should go by domain instead of database name, to be consistent with semi_automated.yml and rfx_analysis.yml